custom_components/koolnova_bms/koolnova/const.py

```diff
@@ -77,7 +77,7 @@ def __int__(self):
 STEP_TEMP_ORDER = 0.5
 
 # Temperature maximale : 50°C
-MAX_TEMP = 50.0
+MAX_TEMP = 35.0
```

Comment:
Hi Mauro,

I've checked your error and fix. Unfortunately, the parameter MAX_TEMP is not used for the engine order temp. 
the official documentation says : 
"_Registers 40069, 70, 71 and 72 (Read Only) : Machine set temperature. Decimal value from 30 to 60 which represents double the set temperature assigned to the AC1, AC2, AC3 and AC4 machines._" that's why in the device.py the value are hardcoded (Min: 30/2 = **15°C** and Max: 60/2 = **30°C**). 
the `async_engine_order_temp` function in operation.py called by the coordinator retreive temperature value from modbus and return it divided by 2.
With your log, we see that the return value (=32.5°C) is greater than the maximum value (30°C) defined in the documentation.
The official documentation isn't correct ? maybe...

Reply:
nice! yes at least on my machine I can go up to 35, I knew about the values from 30 to 60 because those are the values they send over mqtt, but didn't knew about the MAX_TEMP. On my installation it fixed the gauge max and min which could cause the issue, just that. I will revert those consts!